workbench-core/accounts/src/utilities/hostingAccountLifecycleService.ts

```diff
@@ -14,6 +14,14 @@ import _ from 'lodash';
 import { HostingAccountStatus } from '../constants/hostingAccountStatus';
 import AccountService from '../services/accountService';
 
+export class InvalidArtifactBucketArnError extends Error {
```

Comment:
This would be the first time we add custom error classes. Before we start adding these classes, maybe you can draft a one pager to inform the team on when we should create custom error classes and how they're used. 

Some questions I had in mind.
1. What type of errors deserve their own custom error classes?
2. What does a custom error class provide over the Boom error classes?
3. When should the custom error classes go into a specific package versus in `workbench-core-base`? Other packages might want to use the same custom error classes.

Reply:
Sounds good, I'll remove this for now and write something up!